workspaces/marketplace/plugins/marketplace/src/api/MarketplaceBackendClient.tsx

```diff
@@ -36,7 +37,7 @@ export class MarketplaceBackendClient implements MarketplaceApi {
     this.fetchApi = options.fetchApi;
   }
 
-  async getPlugins(): Promise<MarketplacePlugin[]> {
+  async getPlugins(): Promise<MarketplacePluginWithPageInfo> {
```

Comment:
This should use the request parameters and apply all the parameters (filter, order, ...) to the URL below.

Please use `URL` or `URLSearchParams` to escape the values.